src/rapid_pro_to_engagement_db/rapid_pro_to_engagement_db.py

```diff
@@ -95,12 +95,12 @@ def _engagement_db_has_message(engagement_db, message):
     """
     # TODO: Rapid Pro has a bug where timestamps occasionally drift by 1us when runs are archived.
     #       Confirm this is resolved before entering production with an '==' check on timestamps.
-    filter = lambda q: q \
+    messages_filter = lambda q: q \
```

Comment:
We can make this a bit more descriptive to capture what it's querying/checking for. matching_messages_filter.

Reply:
Updated, thanks.